modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/solutions-search-results/ddm-template.ftl

```diff
@@ -103,22 +104,10 @@
 	</#list>
 </#if>
 
-<#assign
-	pageSize = pageSize?has_content?then(pageSize, 15)
-	page = page?has_content?then(page, 1)
-	taxonomyVocabularyName = "Marketplace Product Type"
-	categoryName = "Solution"
-	taxonomyVocabulary = restClient.get("/headless-admin-taxonomy/v1.0/sites/${themeDisplay.getCompanyGroupId()}/taxonomy-vocabularies?fields=id&filter=name eq '${taxonomyVocabularyName}'").items
-	vocabularyCategory = restClient.get("/headless-admin-taxonomy/v1.0/taxonomy-vocabularies/${taxonomyVocabulary[0].id}/taxonomy-categories?fields=id&filter=name eq '${categoryName}'").items
-	productsList = restClient.get("/headless-commerce-admin-catalog/v1.0/products?filter=categoryIds/any(params:params eq '${vocabularyCategory[0].id}')&pageSize=" + pageSize + "&page=" + page)
-	numberFilteredProducts = 0
-	filterCategoriesByUrlParams = getFilterByUrlParams(siteURL)
-/>
+<#assign pageSize = pageSize?has_content?then(pageSize, 15)page = page?has_content?then(page, 1)taxonomyVocabularyName = "Marketplace Product Type" categoryName = "Solution" taxonomyVocabulary = restClient.get("/headless-admin-taxonomy/v1.0/sites/${themeDisplay.getCompanyGroupId()}/taxonomy-vocabularies?fields=id&filter=name eq '${taxonomyVocabularyName}'").items vocabularyCategory = restClient.get("/headless-admin-taxonomy/v1.0/taxonomy-vocabularies/${taxonomyVocabulary[0].id}/taxonomy-categories?fields=id&filter=name eq '${categoryName}'").items productsList = restClient.get("/headless-commerce-admin-catalog/v1.0/products?filter=categoryIds/any(params:params eq '${vocabularyCategory[0].id}')&pageSize=" + pageSize + "&page=" + page)numberFilteredProducts = 0 filterCategoriesByUrlParams = getFilterByUrlParams(siteURL) />
```

Comment:
Hey @welitonpe are you sure the assign variables should be all at the same line? I believe this is a bit confusing. What do you think?